jenkins/databricks/install_deps.py

```diff
@@ -0,0 +1,217 @@
+                 f'{prefix_ws_sp_mvn_hadoop}--org.apache.commons--commons-lang3--org.apache.commons__commons-lang3__*.jar'),
+        Artifact('org.json4s', f'json4s-ast_{scala_version}',
+                 f'{prefix_ws_sp_mvn_hadoop}--org.json4s--json4s-ast_{scala_version}--org.json4s__json4s-ast_{scala_version}__*.jar'),
+        Artifact('org.json4s', f'json4s-core_{scala_version}',
+                 f'{prefix_ws_sp_mvn_hadoop}--org.json4s--json4s-core_{scala_version}--org.json4s__json4s-core_{scala_version}__*.jar'),
+        Artifact('org.javaassist', 'javaassist',
+                 f'{prefix_ws_sp_mvn_hadoop}--org.javassist--javassist--org.javassist__javassist__*.jar'),
+        Artifact('com.fasterxml.jackson.core', 'jackson-core',
+                 f'{prefix_ws_sp_mvn_hadoop}--com.fasterxml.jackson.core--jackson-databind--com.fasterxml.jackson.core__jackson-databind__*.jar'),
+        Artifact('com.fasterxml.jackson.core', 'jackson-annotations',
+                 f'{prefix_ws_sp_mvn_hadoop}--com.fasterxml.jackson.core--jackson-annotations--com.fasterxml.jackson.core__jackson-annotations__*.jar'),
+        Artifact('org.apache.spark', f'spark-avro_{scala_version}',
+                 f'{spark_prefix}--vendor--avro--avro-*.jar'),
+        Artifact('org.apache.avro', 'avro-mapred',
+                 f'{prefix_ws_sp_mvn_hadoop}--org.apache.avro--avro-mapred--org.apache.avro__avro-mapred__*.jar'),
+        Artifact('org.apache.avro', 'avro',
+                 f'{prefix_ws_sp_mvn_hadoop}--org.apache.avro--avro--org.apache.avro__avro__*.jar'),
+    ]
+
+    # log4j-core
+    if spark_version.startswith('3.3'):
+        deps += Artifact('org.apache.logging.log4j', 'log4j-core',
+                         f'{prefix_ws_sp_mvn_hadoop}--org.apache.logging.log4j--log4j-core--org.apache.logging.log4j__log4j-core__*.jar'),
+
+    return deps
+
+def install_deps(deps, spark_version_to_install_databricks_jars, m2_dir, jar_dir):
+    pom_xml_header = """<?xml version=\"1.0\" encoding=\"UTF-8\"?>
+        <project xmlns=\"http://maven.apache.org/POM/4.0.0\"
+            xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"
+            xsi:schemaLocation=\"http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd\">
+            <modelVersion>4.0.0</modelVersion>
+            <groupId>com.nvidia</groupId>
+            <artifactId>rapids-4-spark-databricks-deps-installer</artifactId>
+            <description>bulk databricks deps installer</description>
+            <version>${SPARK_PLUGIN_JAR_VERSION}</version>
+            <packaging>pom</packaging>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-install-plugin</artifactId>
+                        <version>2.4</version>
+                        <executions>
+        """
+    print(pom_xml_header)
```

Comment:
Would it be better to write it out to a file?  generally things should not be printed out by libraries/etc, but it might be cleaner to just include it as an option instead of redirecting the output.